index.html

```diff
@@ -12,32 +12,187 @@
+                        <li class="project-list">ruby</li>
+                    </ul>
+                </div>
+                <button class="button card-button rmv" type="button">See Project</button>
             </li>
-            <li><img src="images/vector.png8.png " alt="my logo " class="social-logo"></li>
-            <li><img src="images/vector.png4.png " alt="my logo " class="social-logo"></li>
-            <li><img src="images/vector.png2.png " alt="my logo " class="social-logo"></li>
         </ul>
     </section>
+    <section class="about-section">
+        <h2 class="about-title">About me</h2>
+        <div class="about">
+            <p class="about-text">Hello I’m a software developer! I can help you build a product ,<br> feature or website Look through some of my work and experience! <br>If you like what you see and have a project you need coded, <br>don’t hestiate to contact me.</p>
+            <button class="button about-button" type="button">Get My Resume</button>
+        </div>
+         <div><img src="images/Vector4.png" alt="line logo" class="line"></div>
+            <ul class="language">
+            <li><img src="images/Rectangle55.png" alt="my logo" class="img">Language
+                <ul>
+                    <li class="language-list">Javascript</li>
+                    <li class="language-list">Ruby</li>
+                    <li class="language-list">Html</li>
+                    <li class="language-list">css</li>
+                </ul>
+            </li>
+            <li>
+                <img src="images/Rectangle56.png" alt="logo" class="img" />Framework
+                <ul>
+                    <li class="language-list">bootstrap</li>
+                    <li class="language-list">Ruby on Rails</li>
+                    <li class="language-list">Capybara</li>
+                    <li class="language-list">selenium</li>
+                </ul>
+            </li>
+            <li>
+                <img src="images/rectangle57.png" alt="logo" class="img"/> Skills
+                <ul>
+                    <li class="language-list">codekit</li>
+                    <li class="language-list">Github</li>
+                    <li class="language-list">Gitlab</li>
+                    <li class="language-list">Terminal</li>
+                </ul>
+            </li>
+        </ul>
+    </section>
+    <section>
+        <h3 class="intro-text">I am always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+        <div class="container"></div>
+        <form class="form" action="https://formspree.io/f/xzbokwoy" method="post">
+            <input type="text" id="fname" name="name" placeholder="Full Name" maxlength="30" required>
+            <input id="email" name="email" type="email" placeholder="Email Address" required>
+            <textarea class="textarea" name="message" maxlength="500" required>Enter text here
```

Comment:
- [ ] It would be great if you replace the text from this textarea and use a placeholder instead, so that way you don't have a custom value already on the textarea once you get into the page. So please, work on this issue.

Reply:
ok thanks